_translator-files/po/de/Running-a-Server.po

```diff
@@ -45,123 +47,125 @@ msgid ""
-msgstr ""
+msgstr "Normalerweise liegen die Probleme auf der _Client_-Seite und sollten dort behoben werden. Wirf einen Blick auf die [Fehlersuche-Seite](/wiki/Client-Troubleshooting)."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:36
 msgid "However, various problems can also arise when setting up Servers - especially when run on a low-bandwidth home connection. It's usually fine to have less than 5 players on a slower-speed home connection (eg 10 Mbit/s down and 1 Mbit/s up). You can read more about network requirements at [different quality settings here](Server-Bandwidth)."
-msgstr ""
+msgstr "Bei der Einrichtung von Servern können jedoch auch verschiedene Probleme auftreten, insbesondere wenn sie über einen Heimanschluss mit geringer Bandbreite betrieben werden. Normalerweise ist das in Ordnung wenn weniger als 5 Teilnehmer verbunden sind. (z.B. 10 Mbit/s down und 1 Mbit/s up). Weitere Informationen zu den Netzwerkanforderungen findest du unter [verschiedene Qualitätseinstellungen hier](Server-Bandbreite)."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:38
 msgid "Consider using a cloud host, not your home internet connection, to get better ping times if you're having problems."
-msgstr ""
+msgstr "Ziehe in Erwägung, einen Cloud-Host und nicht deine private Internetverbindung zu verwenden, um bessere Ping-Zeiten zu erhalten, wenn Latenz Probleme auftreten."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:41
 msgid "### General notes"
-msgstr ""
+msgstr "### Allgemeine Hinweise"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:43
 msgid "Any Server should have at least 1.6GHz CPU frequency and 1GB RAM"
-msgstr ""
+msgstr "Jeder Server sollte mindestens 1.6GHz CPU Frequenz und 1GB RAM haben"
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:44
 msgid "Running a Server may require you to adjust any firewalls running on or outside of your machine or cloud host."
-msgstr ""
+msgstr "Für den Betrieb eines Servers musst du möglicherweise Firewalls anpassen, die auf oder außerhalb deines Rechners oder Cloud-Hosts laufen."
 
 #. type: Plain text
 #: ../wiki/en/Running-a-Server.md:45
 msgid "Running an Unregistered Server at home will require you to [port forward](#port-forwarding) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode."
-msgstr ""
+msgstr "Wenn du zu Hause einen nicht registrierten/privaten Server betreibst, musst du auf deinem Router eine [Portweiterleitung](#running-a-private-server) einrichten. Wenn du einen öffentlichen Server betreibst, sollte eine Portweiterleitung in den meisten Fällen nicht notwendig sein, aber es ist ratsam, dies zu tun, da einige Netzwerke mit Jamulus im Standardmodus möglicherweise nicht richtig funktionieren."
```

Comment:
Not sure why `#running-a-private-server` is there but it should be a translation of `#port-forwarding`

Reply:
Corrected